### ISSUE TITLE
Compiling to xavs2 fails when running Windows 32-bit compilation.

### ISSUE THREAD

**Issue by marierose147** [rdp/ffmpeg-windows-build-helpers#474]
I compiled at Github Actions.
 I tried it with both Ubuntu 20.04 and Ubuntu 18.04 environments.
I repeated the compilation three times, and each time it failed to compile xavs2.
I was able to successfully compile Windows 64-bit with Ubuntu 20.04.
The settings used for compiling are: `./cross_compile_ffmpeg.sh --ffmpeg-git-checkout-version=n4.3.1 --disable-nonfree=n --compiler-flavors=win32 --build-ffmpeg-static=y --build-ffmpeg-shared=n`
[logs download](https://www.dropbox.com/sh/0jdrfv68svcbv5j/AAASE6JyJEcXpsOF6rA0fnoVa?dl=0)

**Comment by AnimMouse** [comment]
Me too. Still failing at my [GitHub Actions](https://github.com/AnimMouse/ffmpeg-stable-autobuild/actions/runs/182317010).
```
2020-07-25T05:54:28.0229346Z make: *** [encoder/aec_ctx.o] Error 1
2020-07-25T05:54:28.0229817Z Makefile:265: recipe for target 'encoder/aec_ctx.o' failed
```

**Comment by AnimMouse** [comment]
#475

**Comment by AnimMouse** [comment]
It is now [working](https://github.com/AnimMouse/ffmpeg-stable-autobuild/actions/runs/183302044), fixed by #475. Thanks!
